codegen/module.go

```diff
@@ -1657,12 +1672,16 @@ type PackageInfo struct {
 	// IsExportGenerated is true if the export type is provided by the
 	// generated package, otherwise it is assumed that the export type resides
 	// in the non-generated package
-	IsExportGenerated bool
+	IsExportGenerated    bool
+	CustomInitialisation bool
```

Comment:
Can you please document how a developer should read `CustomInitialisation`  in the context of a module?